Pluralize content view to view(s) (#3801)
theforeman/foreman-documentation: guides/common/modules/ref_best-practices-for-content-views.adoc

* Content views that bundle content, such as {client-os} and additional software like `Apache-2.4` or `PostgreSQL-16.2`, are easier to maintain.
Content views that are too small require more maintenance.
* If you require daily updated content, use the content view `Default Organization View`, which contains the latest synchronized content from all repositories and is available in the Library lifecycle environment.
* To give hosts access to content from multiple content views, such as when you update one content view weekly and another monthly, you have two options:

> - Assign multiple content view environments to hosts. 

Reviewer: ```suggestion
** Assign multiple content view environments to hosts. 
```

For more information, see xref:assigning-content-view-environments-to-hosts[].

> - Assign composite content views to hosts. 

Reviewer: ```suggestion
** Assign composite content views to hosts. 
```

> - Assign composite content view environments to hosts. 

Reviewer 2: "_Composite content view environments_" isn't a thing. But "_a content view environment which contains a composite content view_" is way too wordy, so I say we just go with the original.
```suggestion
- Assign composite content views to hosts. 
```

* Content views that bundle content, such as {client-os} and additional software like `Apache-2.4` or `PostgreSQL-16.2`, are easier to maintain.
Content views that are too small require more maintenance.
* If you require daily updated content, use the content view `Default Organization View`, which contains the latest synchronized content from all repositories and is available in the Library lifecycle environment.

> * Restrict composite content views to situations that require greater flexibility, for example, if you update one content view weekly, another monthly, or assign hosts to multiple content view environments.

Reviewer 2: Assigning multiple content view environments is an _alternative_ to using composite content views. It should be its own item.

```suggestion
* To give hosts access to content from multiple content views, for example, if you update one content view on a weekly basis and another content view on a monthly basis, you have two options:
- Assign hosts to multiple content view environments. For more info see xxxxx
- Assign hosts to composite content views. For more info see xxxxx
```

Author: We don't have a section on assigning composite content views to hosts. I can reference the concept module `Comparison of content view environments and composite content views`.

Reviewer 2: sounds good! :+1:
